influxql/parser.go

```diff
@@ -2688,11 +2688,13 @@ func (p *Parser) scan() (tok Token, pos Pos, lit string) { return p.s.Scan() }
 
 // scanIgnoreWhitespace scans the next non-whitespace token.
 func (p *Parser) scanIgnoreWhitespace() (tok Token, pos Pos, lit string) {
```

Comment:
I updated the comment, but I'm having difficulty thinking of a name that isn't too long for this function. I personally consider comments to be whitespace, but I would be willing to change the name as long as it's not `scanIgnoreWhitespaceAndComments()`.